lib/resourcebuilder/apps.go

```diff
@@ -72,6 +76,34 @@ func (b *deploymentBuilder) Do(ctx context.Context) error {
 		}
 	}
 
+	// the CVO is the only operator which doesn't have knowledge of its own operand (itself).  If we detect the CVO deployment
```

Comment:
Can we drop this opening sentence?  It doesn't seem relevant to me, and the CVO has multiple operands (all the manifests supplied in the release image), of which the CVO's own deployment is one.

Reply:
sure

Reply:
f0f00ff9b9b no longer contains the line.  Marking this thread resolved.